core/iwasm/fast-jit/jit_ir.c

```diff
@@ -1558,3 +1558,43 @@ _jit_insn_check_opnd_access_LookupSwitch(const JitInsn *insn)
     unsigned opcode = insn->opcode;
     return (insn_opnd_kind[opcode] == JIT_OPND_KIND_LookupSwitch);
 }
+
+#if defined(BUILD_TARGET_X86_64) || defined(BUILD_TARGET_AMD_64)
+bool
+jit_lock_regs_in_insn(JitCompContext *cc, JitInsn *the_insn, JitReg locked_reg)
+{
+    bool ret = false;
+    JitInsn *prevent_spill = NULL;
+    JitInsn *indicate_using = NULL;
+
+    if (!the_insn)
+        goto just_return;
+
+    if (jit_cc_is_hreg_fixed(cc, locked_reg)) {
+        ret = true;
+        goto just_return;
+    }
+
+    /* give the virtual register of the locked hard register a minimum, non-zero
+     * distance, and prevent from being spilled out */
```

Comment:
How about changing `and prevent from being spilled out` to `so as to prevent it from being spilled out`?